force-app/main/default/classes/CRLP_Batch_Base.cls

```diff
@@ -503,6 +506,9 @@ public abstract class CRLP_Batch_Base {
             get {
                 if (maxDaysAgo == null) {
                     maxDaysAgo = getMaxIntegerValueFromRollupsForType(CRLP_Operation.TimeBoundOperationType.Days_Back);
+                    if (maxDaysAgo == null && base.customizableRollupSettings.LimitRecalculatedRecurringDonations__c) {
+                        maxDaysAgo = (Integer) base.customizableRollupSettings.RecurringDonationLastNDays__c;
```

Comment:
Is there a possibility that if `RecurringDonationLastNDays__c` is null this would cause an NPE?